### PR TITLE
Final version of logout popup component

### PR DESCRIPTION
**Pull request by Nazaqat**

### CODE REVIEW

File: src/client/components/logout/logout.component.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    </div>
+  );
+};
+
+const LogoutContent = ({ children }) => {
+  return (
+    <>
+      <div className="content">
+        <p>
+          Hi {userName} <br />
```
Comment (bytewiz) [inline src/client/components/logout/logout.component.js:47]: `userName` should be a prop, not a hardcoded string

File: src/client/components/logout/logout.component.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        </p>
+      </div>
+      {children}
+    </>
+  );
+};
+
+const LogoutActions = ({ closeAction }) => (
+  <div className="actions">
+    <button className="confirm-logout" type="submit" onClick="{() => {}}">
```
Comment (bytewiz) [inline src/client/components/logout/logout.component.js:58]: Shouldn't you pass a "logout function" here as a prop?

File: src/client/components/logout/logout.component.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Popup from 'reactjs-popup';
+import './logout.style.css';
+import PropTypes from 'prop-types';
+
+const userName = 'Servet';
```
Comment (bytewiz) [inline src/client/components/logout/logout.component.js:6]: Don't hardcode this :)

File: src/client/components/logout/logout.component.js (hunk 2 above)
Comment (senner008) [reply src/client/components/logout/logout.component.js:58]: It's unknown by now

Comment (bytewiz) [reply src/client/components/logout/logout.component.js:58]: Alright, it is fine to keep like this but would rather have it as a prop and pass an "empty function" here. @senner008 what is your opinion on this? Then we shouldn't have to change the file again 🤞

Comment (senner008) [reply src/client/components/logout/logout.component.js:56]: you decide...

**Comment by Nazaqat** [comment]
> Not sure what's going on, but you're deleting the knexfile and adding some sqlite files?

Dont know, how it happened but sqlite file has been deleted and knex.js has been added.

**Comment by Nazaqat** [comment]
new branch has been created called feature/logout-component

**Pull request closed without merging**